src/CalculatorUnitTests/HistoryTests.cpp

```diff
@@ -42,15 +42,15 @@ namespace CalculatorFunctionalTests
         HistoryViewModel^ m_historyViewModel;
         StandardCalculatorViewModel^ m_standardViewModel;
 
-        void Initialize(unsigned int windowId = 0)
+        void Initialize(unsigned int /*windowId*/ = 0)
```

Comment:
If this parameter isn't used, let's just remove it entirely.

Reply:
I wasn't sure if it was needed as part of the test framework.  It looks like it's not needed, so I'll remove it.